stdlib/format.mli

```diff
@@ -493,9 +493,11 @@ coupled variables, margin and maximum indentation limit.
 *)
 
 type geometry = { max_indent:int; margin: int}
+(** @since 4.08.0 *)
 
 val check_geometry: geometry -> bool
-(** Check if the formatter geometry is valid: [1 < max_indent < margin] *)
+(** Check if the formatter geometry is valid: [1 < max_indent < margin] 
+     @since 4.08.0 *)
```

Comment:
(You might wish to fix the misalignment between `since` and `Check`)

Reply:
Ah, missed it! Fixed it now.